pravega-client-examples/README.md

```diff
@@ -140,3 +140,53 @@ To demonstrate the use of `StreamCut`s, we provide a CLI. To use it, please exec
 $ bin/streamCutsCli [-scope myScope] [-name myStream] [-uri tcp://127.0.0.1:9090]
 ```
 
+## `secure`
+This example includes two applications `SecureReader` and `SecureWriter`. These applications illustrate
+how to setup clients connecting to a security-enabled Pravega cluster. Specifically, they show how to:
+* Enable SSL/TLS (HTTPS) communications with a Pravega cluster for data-in-transit encryption and server authentication.
+* Pass credentials to a Pravega cluster for client authentication and authorization.
+
+### Execution
+
+First, ensure that the Pravega cluster that the applications are going to work with, has both TLS and
+Auth (authentication and authorization) enabled. See Pravega documentation for how to enable TLS and Auth.
+
+Now, you might want to run `SecureWriter` in one window and `SecureReader` in another window.
```

Comment:
We are assuming here that the scope/stream to be used will have to be created outside the scope of this sample, right?

Reply:
No, the scope and stream are created inside the program. 

https://github.com/pravega/pravega-samples/blob/5cd3ea7bb83cbdf986032a8002f0fd47ad1c2406/pravega-client-examples/src/main/java/io/pravega/example/secure/SecureWriter.java#L116

https://github.com/pravega/pravega-samples/blob/5cd3ea7bb83cbdf986032a8002f0fd47ad1c2406/pravega-client-examples/src/main/java/io/pravega/example/secure/SecureWriter.java#L124